*** ../bb.sh	2016-02-07 20:33:05.597544731 -0300
--- ../../bashblog/bb.sh	2016-02-07 02:03:26.986386315 -0300
***************
*** 880,886 ****
          pubdate=$(LC_ALL=C date +"%a, %d %b %Y %H:%M:%S %z")
          echo '<?xml version="1.0" encoding="UTF-8" ?>' 
          echo '<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">' 
!         echo "<channel><title>$global_title</title><link>$global_url/$index_file</link>"
          echo "<description>$global_description</description><language>en</language>"
          echo "<lastBuildDate>$pubdate</lastBuildDate>"
          echo "<pubDate>$pubdate</pubDate>"
--- 880,886 ----
          pubdate=$(LC_ALL=C date +"%a, %d %b %Y %H:%M:%S %z")
          echo '<?xml version="1.0" encoding="UTF-8" ?>' 
          echo '<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">' 
!         echo "<channel><title>$global_title</title><link>$global_url</link>"
          echo "<description>$global_description</description><language>en</language>"
          echo "<lastBuildDate>$pubdate</lastBuildDate>"
          echo "<pubDate>$pubdate</pubDate>"
***************
*** 914,920 ****
  # generate headers, footers, etc
  create_includes() {
      {
!         echo "<h1 class=\"nomargin\"><a class=\"ablack\" href=\"./$index_file\">$global_title</a></h1>" 
          echo "<div id=\"description\">$global_description</div>"
      } > ".title.html"
  
--- 914,920 ----
  # generate headers, footers, etc
  create_includes() {
      {
!         echo "<h1 class=\"nomargin\"><a class=\"ablack\" href=\"$global_url\">$global_title</a></h1>" 
          echo "<div id=\"description\">$global_description</div>"
      } > ".title.html"
  
